lib/new_relic/agent/instrumentation/action_view_subscriber.rb

```diff
@@ -61,12 +58,15 @@ def recordable?(event_name, metric_name)
         RENDER_TEMPLATE_EVENT_NAME = 'render_template.action_view'.freeze
         RENDER_PARTIAL_EVENT_NAME = 'render_partial.action_view'.freeze
         RENDER_COLLECTION_EVENT_NAME = 'render_collection.action_view'.freeze
+        RENDER_LAYOUT_EVENT_NAME = 'render_layout.action_view'.freeze
 
         def metric_action(name)
           case name
           when /#{RENDER_TEMPLATE_EVENT_NAME}$/o then 'Rendering'
           when RENDER_PARTIAL_EVENT_NAME then 'Partial'
           when RENDER_COLLECTION_EVENT_NAME then 'Partial'
+          when RENDER_LAYOUT_EVENT_NAME then 'Layout'
+          else 'Unknown'
```

Comment:
There are places where "Unknown" (capital U) are used, but for the subscriber stuff we've been consistently using "unknown" (lower case u). Perhaps this should be changed?

Reply:
Great catch. I wonder if we should define the `UNKNOWN` constant in NotificationsSubscriber so we have the same UNKNOWN everywhere?